source/common/buffer/buffer_impl.cc

```diff
@@ -217,21 +638,69 @@ OwnedImpl::OwnedImpl(const Instance& data) : OwnedImpl() { add(data); }
 OwnedImpl::OwnedImpl(const void* data, uint64_t size) : OwnedImpl() { add(data, size); }
 
 std::string OwnedImpl::toString() const {
-  uint64_t num_slices = getRawSlices(nullptr, 0);
-  STACK_ARRAY(slices, RawSlice, num_slices);
-  getRawSlices(slices.begin(), num_slices);
-  size_t len = 0;
-  for (const RawSlice& slice : slices) {
-    len += slice.len_;
+  if (old_impl_) {
```

Comment:
Does this method need if? Seems same and not depends on old/new impl?

Reply:
Fixed in the latest update